core/src/main/scala/org/apache/spark/deploy/worker/DriverRunner.scala

```diff
@@ -57,7 +57,8 @@ private[deploy] class DriverRunner(
   @volatile private[worker] var finalException: Option[Exception] = None
 
   // Timeout to wait for when trying to terminate a driver.
-  private val DRIVER_TERMINATE_TIMEOUT_MS = 10 * 1000
+  private val DRIVER_TERMINATE_TIMEOUT_MS =
+    conf.getTimeAsMs("spark.worker.driverTerminateTimeout", "10s")
```

Comment:
Just wondering if maybe adding something to the property to be clear that this is for a driver with deploy mode cluster only?  Although it is prefixed with `worker` so maybe that is good enough.